tools/dataProtector/dataProtectorCore/processProtectedData.md

```diff
@@ -116,12 +119,19 @@ A set of requester secrets necessary for the application's execution. This is
 represented as a mapping of numerical identifiers to corresponding secrets
 stored in the secrets manager needed for the application's execution.
 
+Secrets are accessible during the application's execution with environment
+variables `IEXEC_REQUESTER_SECRET_n`.
+
 ```js
 const taskid = await dataProtector.processProtectedData({
   protectedData: '0x123abc...',
   app: '0xC2E...',
   maxPrice: 10,
-  secrets: ['secret1', 'secret2'], // [!code focus]
+  secrets: {
+    // [!code focus]
+    1: 'secret1',
+    2: 'secret2',
+  },
```

Comment:
Problem here with the code focus, the parameter is not highlighted:

<img width="718" alt="Capture d’écran 2024-04-25 à 16 52 03" src="https://github.com/iExecBlockchainComputing/documentation-tools/assets/4280765/ad166ea6-6144-4529-86d4-43c54d991735">

The code comment `// [!code focus]` needs to be repeated for each line.

And unfortunately we also need to ignore Prettier here to keep the correct code highlight.

```
<!-- prettier-ignore-start -->
```js
const taskid = await dataProtector.processProtectedData({
  protectedData: '0x123abc...',
  app: '0xC2E...',
  maxPrice: 10,
  secrets: { // [!code focus]
    1: 'secret1', // [!code focus]
    2: 'secret2', // [!code focus]
  }, // [!code focus]
});
```
<!-- prettier-ignore-end -->
```